apps/family-public-law/fpl-cron-ccd-data-migration-tool/fpl-cron-ccd-data-migration-tool.yaml

```diff
@@ -10,6 +10,17 @@ spec:
     job:
       useInterpodAntiAffinity: true
       image: hmctspublic.azurecr.io/fpl/cron-ccd-data-migration-tool:prod-f597322-20230321171332 #{"$imagepolicy": "flux-system:fpl-cron-ccd-data-migration-tool"}
+      schedule: "30 12 30 2 *"
+      cpuRequests: "1000m"
+      cpuLimits: "2500m"
+      memoryRequests: "1024Mi"
```

Comment:
THis is going into all environments yes?

Reply:
Demo doesn't need this.

Reply:
You would need to create some new patch files per environment 

example; https://github.com/hmcts/cnp-flux-config/tree/master/apps/cnp/plum-frontend

& kustomized to the likes of AAT - like this, https://github.com/hmcts/cnp-flux-config/blob/master/apps/cnp/aat/base/kustomization.yaml#L8

Reply:
I have changed as per shared example.